inst/json/intervention_options.json

```diff
@@ -156,7 +156,7 @@
+            "documentation": "<p>The price of different vector control interventions will vary so can be defined by the user in $USD. For simplicity, it is assumed that there is a linear relationship between cost and population coverage, we do not consider inflation in this iteration of the tool.</p><strong>Price of standard ITN ($USD)</strong><p>Price per pyrethroid-only ITN (expected to be replaced every 3-years).</p><strong>Price of Pyrethroid-PBO ITN ($USD)</strong><p>Price per Pyrethroid-PBO ITN (expected to be replaced every 3-years).</p><strong>ITN mass distribution campaign delivery cost per person ($USD)</strong><p>Cost to deliver nets to each person (equivalent for each ITN type). enough nets are provided to match the number of people per net and the procurement buffer.</p><strong>Price of spray product per person (annual $USD)</strong><p>The price per person of long-lasting IRS product averaged for each year. Include the average cost for both the IRS product and implementation of IRS. If different IRS products are used in different years, please average the product costs and provide an annual cost per person protected by IRS (in $USD)</p><strong>Total available budget ($USD)</strong><p>The total available budget to cover the next 3-years, is required to assess the most feasible intervention options across zones. Cost effectivessness can be optimised across zones.</p><strong>Zonal budget ($USD)</strong><p>The decision makers can choose to cap the budget for each zone to some specified maximum. Please enter the budget for this zone.</p>"
```

Comment:
we should make these markdown too at some point. I can open a ticket if you agree

Reply:
Yeah I agree.